libc/src/__support/wchar/character_converter.h

```diff
@@ -29,8 +30,8 @@ class CharacterConverter {
   int push(char8_t utf8_byte);
   int push(char32_t utf32);
```

Comment:
I note that pushing a utf8 byte onto a state that hasn't been completely popped causes an error (in Sriya's implementation), but pushing a utf32 value onto a state that hasn't been completely popped doesn't error.  It would be good to be consistent with this, I think?